packages/integrations/mdx/README.md

```diff
@@ -71,413 +66,88 @@ Finally, restart the dev server.
-  <p>A blockquote with <em>some</em> emphasis.</p>
-</blockquote>
-```
-
-But what if you want to specify your own markup for these blockquotes? In the above example, you could create a custom `<Blockquote />` component (in any language) that either has a `<slot />` component or accepts a `children` prop.
-
-```astro title="src/components/Blockquote.astro"
----
-const props = Astro.props;
----
-
-<blockquote {...props} class="bg-blue-50 p-4">
-  <span class="text-4xl text-blue-600 mb-2">“</span>
-  <slot />
-</blockquote>
-```
-
-Then in the MDX file you import the component and export it to the `components` export.
-
-```mdx title="src/pages/posts/post-1.mdx" {2}
-import Blockquote from '../components/Blockquote.astro';
-export const components = { blockquote: Blockquote };
-```
+Visit the [MDX docs](https://mdxjs.com/docs/what-is-mdx/) to learn about using standard MDX features.
 
-Now, writing the standard Markdown blockquote syntax (`>`) will use your custom `<Blockquote />` component instead. No need to use a component in Markdown, or write a remark/rehype plugin! Visit the [MDX website](https://mdxjs.com/table-of-components/) for a full list of HTML elements that can be overwritten as custom components.
-
-#### Custom components with imported `mdx`
-
-When rendering imported MDX content, custom components can be passed via the `components` prop.
-
-Note: An MDX file's exported components will _not_ be used unless you manually import and pass them via the `components` property. See the example below:
-
-```astro title="src/pages/page.astro" "components={{...components, h1: Heading }}"
----
-import { Content, components } from '../content.mdx';
-import Heading from '../Heading.astro';
----
-
-<Content components={{...components, h1: Heading }} />
-```
-
-### Syntax highlighting
-
-The MDX integration respects [your project's `markdown.syntaxHighlight` configuration](https://docs.astro.build/en/guides/markdown-content/#syntax-highlighting).
-
-We will highlight your code blocks with [Shiki](https://github.com/shikijs/shiki) by default. You can customize this highlighter using the `markdown.shikiConfig` option in your `astro.config`. For example, you can apply a different built-in theme like so:
-
-```js
-// astro.config.mjs
-export default {
-  markdown: {
-    shikiConfig: {
-      theme: 'dracula',
-    },
-  },
-  integrations: [mdx()],
-}
-```
-
-Visit [our Shiki configuration docs](https://docs.astro.build/en/guides/markdown-content/#shiki-configuration) for more on using Shiki with Astro.
-
-#### Switch to Prism
-
-You can also use the [Prism](https://prismjs.com/) syntax highlighter by setting `markdown.syntaxHighlight` to `'prism'` in your `astro.config` like so:
+## Configuration
 
-```js
-// astro.config.mjs
-export default {
-  markdown: {
-    syntaxHighlight: 'prism',
-  },
-  integrations: [mdx()],
-}
-```
+Once the MDX integration is installed, no configuration is necessary to use `.mdx` files in your Astro project.
 
```

Comment:
Slightly left-field idea you should feel free to ignore but given the extensive detailed explanation for some of these options a mini table of contents here might be helpful. We have this in the sidebar anyway on the docs site, so not relevant there, but might help navigation on npm/GitHub.

```suggestion

- [`extendPlugins`](#extendplugins)
- [`remarkRehype`](#remarkrehype)
- [`remarkPlugins`](#remarkplugins)
- [`rehypePlugins`](#rehypeplugins)
- [`recmaPlugins`](#recmaplugins)

```